### PR TITLE
send news and adoption change notification with delay

### PR DESCRIPTION
**Pull request by ice-cronus**

### CODE REVIEW

File: notifications/contract.go [hunk 1]
```diff
@@ -249,7 +249,11 @@ type (
 			Levels []string `yaml:"levels"`
 			Roles  []string `yaml:"roles"`
 		} `yaml:"disabledAchievementsNotifications"`
-		DeeplinkScheme       string                   `yaml:"deeplinkScheme"`
+		DeeplinkScheme     string `yaml:"deeplinkScheme"`
+		NotificationDelays map[push.SubscriptionTopic]struct {
+			MinDelay uint `yaml:"minDelay"`
```
Comment (ice-ares) [inline notifications/contract.go]: just add a sec to em, cuz its confusing

File: notifications/contract.go [hunk 2]
```diff
@@ -249,7 +249,11 @@ type (
...
+			MaxDelay uint `yaml:"maxDelay"`
+		} `yaml:"notificationDelays"`
```
Comment (ice-ares) [inline notifications/contract.go]: mapstructure to make sure notificationDelays is expected

File: application.yaml [hunk 3]
```diff
@@ -104,6 +104,10 @@ notifications: &notifications
       - s1
       - c1
       - l1
+  notificationDelays:
+    news_en_v2:
```
Comment (ice-ares) [inline application.yaml:110]: no, im not gonna add 50 keys there. Think of some way to do it

Comment (ice-cronus) [reply application.yaml:110]: But u said it is by topic... Are we ok with news, system here? Like a entry-prefix

Comment (ice-ares) [reply application.yaml:110]: add defaults for those min max delays in cfg, and then override em with the key present here

**Pull request merged**